.travis.yml

```diff
@@ -12,8 +12,8 @@ env:
   - KAFKA_HOSTNAME=localhost
   - DEBUG=true
   matrix:
-  - KAFKA_VERSION=0.11.0.2
```

Comment:
Our stated support commitment means we shouldn't remove 0.11 from CI until 2 months after 1.1 comes out. If you're looking to trim CI load we'll be able to drop Go 1.8 in a few weeks.

Reply:
Thanks, updated.